### PR TITLE
chore: Temporarily make publish to web apps optional

### PR DESCRIPTION
**Pull request by MartinZikmund**
GitHub Issue (If applicable): #

## PR Type

What kind of change does this PR introduce?

- Bugfix
- Feature
- Code style update (formatting)
- Refactoring (no functional changes, no api changes)
- Build or CI related changes
- Documentation content changes
- Project automation
- Other... Please describe:

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tested on iOS.
- [ ] Tested on Wasm.
- [ ] Tested on Android.
- [ ] Tested on UWP.
- [ ] Tested in both **Light** and **Dark** themes.
- [ ] Associated with an issue (GitHub or internal)

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: .github/workflows/azure-static-webapp.yml [hunk 1]
```diff
@@ -53,6 +53,7 @@ jobs:
 
       - name: Publish to Azure Static WebApps
         id: builddeploy_uno
+        continue-on-error: true
```
Comment (jeromelaban) [inline .github/workflows/azure-static-webapp.yml:56]: Why is this needed now? The PR checks should pass anyways, I removed the requirement for that step.

Comment (MartinZikmund) [reply .github/workflows/azure-static-webapp.yml:56]: Ah, I see, I didn't notice the reason the merging was blocked was approve missing.
